gpu-operator/life-cycle-policy.rst

```diff
@@ -86,60 +86,58 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
      - Version
 
    * - NVIDIA GPU Operator
-     - v24.3.0
+     - v24.6.0
 
    * - NVIDIA GPU Driver
-     - | `550.90.07 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-550-90-07/index.html>`_ (recommended),
-       | `550.54.15 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-550-54-15/index.html>`_ (default),
-       | `535.183.01 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-535-183-01/index.html>`_,
-       | `470.256.02 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-470-256-02/index.html>`_
```

Comment:
@cdesiniotis , @tariq1890  reminded me in the DMA-BUF update that 470 is EOL at the end of this month.  I removed it from this table, but have no objection to restoring it if that's preferred.

Reply:
I think this is fine as-is, thanks!